Resources/views/Security/login_content.html.twig

```diff
@@ -9,14 +9,11 @@
         <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
     {% endif %}
 
-    <label for="username">{{ 'security.login.username'|trans }}</label>
-    <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" />
+    {{ form_row(form._username, {'full_name': '_username', 'label': 'security.login.username'|trans}) }}
```

Comment:
I tried to put full_name in Controller, but it requires extra innecesary code, and text's label is more easy and quickly to edit if it is in the login_content.html.twig